modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/ApplicationModel.java

```diff
@@ -342,6 +340,11 @@ private void executeOnComponentTree(final ComponentModel component, final Compon
         return models;
     }
 
+    private boolean isConfigurationTopComponent(ConfigLine parent)
```

Comment:
Can change this for MuleHierarchicalBeanDefinitionParserDelegate#isMuleRootElement()?